### PR TITLE
Codemods

### PR DESCRIPTION
**Pull request by PlayMa256**
Just for code review purposes... i'll properly add description when it is ready

### CODE REVIEW

**Comment by PlayMa256** [comment]
@diegohaz

**Comment by diegohaz** [comment]
I think we should name the file `v016.js` instead of `v16.js` so it won't conflict when we release v1+

**Comment by PlayMa256** [comment]
By the way. What is a good name to call the cli? I mean, `reakit-codemods param1 param2` <---

**Comment by Thomazella** [comment]
`reakit-codemods foo bar` looks good to me 🤔

File: packages/reakit-codemods/src/transformations/v016.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+export default function transformer(file, api) {
+  const j = api.jscodeshift;
+
+  const ast = j(file.source);
+
+  let disguisedAs = "";
+  let elementItself;
+  let asDeclarationHasArray = false;
+
+  ast.find(j.ImportDeclaration).forEach(importDeclarations => {
```
Comment (PlayMa256) [inline packages/reakit-codemods/src/transformations/v016.js]: reminder: add search for reakit

File: packages/reakit-codemods/src/transformations/v016.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    if (importDeclarations) {
```
Comment (matheus1lva) [inline packages/reakit-codemods/src/transformations/v016.js]:
```suggestion
    if (importDeclarations && importDeclarations.value.source.value === 'reakit') {
```

**Comment by diegohaz** [comment]
Yep

**Comment by PlayMa256** [comment]
Ok, i'll change it!

**Comment by PlayMa256** [comment]
Fixed!
Seems to work around that problem

**Comment by PlayMa256** [comment]
I'll also change the approach on testing. As you see everything is being transformed right, don't merge yet.

**Pull request merged**